scripts/aggregate-changelogs/config.yaml

```diff
@@ -0,0 +1,54 @@
+# Changes and Releases automation config
+#
+# Under '.repositories', each key represents a Github repository
+# which we want to fetch release info from.
+#
+# The 'category' property determines the changes category the
+# entries from the repo land in.
+#
+repositories:
+  giantswarm/releases:
+    # The repo 'giantswarm/releases' gets special treatment, so no
+    # config needed
+  giantswarm/happa:
+    category: Web UI
+  giantswarm/docs:
+    category: Documentation
+  giantswarm/apiextensions:
+    category: Control Plane Kubernetes API
+  giantswarm/api-spec:
+    category: Rest API
+  giantswarm/gsctl:
+    category: gsctl
+  giantswarm/kubectl-gs:
+    category: kubectl gs
+  giantswarm/cert-manager-app:
+    category: Managed Apps
+  giantswarm/metrics-server-app:
+    category: Managed Apps
+  giantswarm/coredns-app:
+    category: Managed Apps
+  giantswarm/external-dns-app:
+    category: Managed Apps
+  giantswarm/cert-exporter:
+    category: Managed Apps
+  giantswarm/chart-operator:
+    category: Managed Apps
+  giantswarm/kube-state-metrics-app:
+    category: Managed Apps
+  giantswarm/net-exporter:
+    category: Managed Apps
+  giantswarm/node-exporter-app:
+    category: Managed Apps
+  giantswarm/cluster-autoscaler-app:
+    category: Managed Apps
+  giantswarm/aqua-app:
+    category: Managed Apps
+  giantswarm/efk-stack-app:
+    category: Managed Apps
+  giantswarm/kong-app:
+    category: Managed Apps
+  giantswarm/nginx-ingress-controller-app:
+    category: Managed Apps
+  giantswarm/prometheus-operator-app:
+    category: Managed Apps
```

Comment:
@cokiengchiara @oponder The goal here is to have all "official" apps included. Can you check whether this is complete? I added those that are pre-installed on Azure, AWS or KVM and those that are in the Giant Swarm Catalog.

Reply:
Looks ok to me for now! Is there any rhyme or reason to the order? Can it be alphabetised for easier spotting of missing ones and changes in the future?

Reply:
You're right, will order by key.